### PR TITLE
Test oak_node module loading code

### PR DESCRIPTION
**Pull request by daviddrysdale**

### CODE REVIEW

File: oak/server/BUILD [hunk 1]
```diff
@@ -76,14 +76,25 @@ cc_library(
...
+        ":testdata_wasm",
+    ],
     deps = [
         ":oak_node",
         "@gtest//:gtest_main",
     ],
 )
 
+# Each file is generated from the corresponding %.wat file with:
+#  wat2wasm %.wat > %.wasm
```
Comment (tiziano88) [inline oak/server/BUILD]: Would it be possible to have a genrule that actually does this transformation (assuming wat2wasm is installed, or, better, downloading a specific version via bazel)?

Comment (daviddrysdale) [reply oak/server/BUILD]: I thought about using a `genrule` but I wasn't keen on imposing a requirement to have a local `wat2wasm` in order to run the tests.  I didn't think of getting Bazel to download and build it itself though &ndash; I'll take a look.

Comment (tiziano88) [reply oak/server/BUILD]: I think downloading a pre compiled binary would suffice -- as long as it works for both Linux and Mac, I guess

Comment (daviddrysdale) [reply oak/server/BUILD]: We already pull in the wabt repository, so getting a `cc_binary` target for `wat2wasm` wasn't too hard, and I had a go at my first .bzl file.

**Pull request merged**